README.md

```diff
@@ -74,6 +75,24 @@ Then, use the script:
 ./make_printable.sh
 ```
 
+### 🔎 Spellchecking
+
+TeXstudio has built-in spellchecking, but the first steps have been made towards automated spellchecking with aspell. For local development, after installing the tool, you can run it from the command line for example with
+
+```bash
+aspell -d en_US -p=./.aspell.homm3.pws --mode=tex --dont-backup check main.tex
+```
+
+or when wanting to check all `.tex` files then with
+
+```bash
+find . -type f -name "*.tex" -exec aspell -d en_US -p=./.aspell.homm3.pws --mode=tex --dont-backup check {} \;
+```
+
+Please note that currently the tool will flag many parameters in LaTeX commands. We are currently looking into it, how best to remediate this.
```

Comment:
```suggestion
Please note that currently the tool will flag many parameters in LaTeX commands.
We are currently looking into it, how best to remediate this.
```
Please keep it one sentence per line.

Reply:
Is `git amend` + `force-push` OK for you for the changes in the README or would you prefer a separate commit?

Reply:
Done.